### PR TITLE
filter in binding

### PR DESCRIPTION
**Pull request by qmacro**
Addressing #5, here is an example of how to influence what data is returned, by adding system query parameters to the binding info.

You can add these system query parameters to the binding info, directly in the aggregation binding expression. See this [Filtering](https://sapui5.hana.ondemand.com/sdk/#/topic/5338bd1f9afb45fb8b2af957c3530e8f.html) section for more information. 

To demonstrated, I decided to use the `Discontinued` boolean property of the `Products` entity type. In this branch I replaced the `StandardListItem` with an `ObjectListItem` control so I could use an `ObjectMarker` to identify discontinued products with a flag, and then extended the binding info to add a `$filter` system query parameter value (`Discontinued eq true`). Change this to `Discontinued eq false` to see the effect for yourself, for example).

In other words, the simplest binding value for the `items` aggregation would be something like this:

```
{/Products}
```

But additional information can be supplied if this binding value is extended to an object, like this:

```
{
  path: '/Products',
  parameters: {
    $filter: 'Discontinued eq true'
  }
}
```

Here's a part of the list display with the filter applied:

screenshot 2022-05-26 at 08 16 25

We can take a brief look at this in [episode 3, on Fri 27 May](https://www.youtube.com/watch?v=Bln2A0_OauY).

### CODE REVIEW

**Comment by qmacro** [comment]
Closing as this was just an example.
